File: CS/BandedViewBandDefaultColumn/BandedViewBandDefaultColumn/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;

namespace BandedViewBandDefaultColumn
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			myGridControl1.DataSource = CreateTable(30);

			((MyBandedGridColumn)myBandedGridView1.Columns["Vendor"]).DefaultBandColumn = true;
			((MyBandedGridColumn)myBandedGridView1.Columns["Model"]).DefaultBandColumn = true;
		}

		private DataTable CreateTable(int RowCount)
		{
			DataTable tbl = new DataTable();
			tbl.Columns.Add("ID", typeof(int));
			tbl.Columns.Add("Vendor", typeof(string));
			tbl.Columns.Add("Model", typeof(string));
			tbl.Columns.Add("ProductNumber", typeof(int));

						
			for ( int i = 0; i < RowCount; i++ )
			{
				int rnd = new Random(i).Next(1, 4);
				tbl.Rows.Add(new object[] { i, String.Format("Vendor {0}", rnd), String.Format("Model {0}", i), i * new Random().Next(1, 10000) });
			}

			return tbl;
		}

	}
}

File: VB/BandedViewBandDefaultColumn/BandedViewBandDefaultColumn/Form1.vb
Imports System
Imports System.Data
Imports System.Windows.Forms

Namespace BandedViewBandDefaultColumn
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            myGridControl1.DataSource = CreateTable(30)

            CType(myBandedGridView1.Columns("Vendor"), MyBandedGridColumn).DefaultBandColumn = True
            CType(myBandedGridView1.Columns("Model"), MyBandedGridColumn).DefaultBandColumn = True
        End Sub

        Private Function CreateTable(ByVal RowCount As Integer) As DataTable
            Dim tbl As New DataTable()
            tbl.Columns.Add("ID", GetType(Integer))
            tbl.Columns.Add("Vendor", GetType(String))
            tbl.Columns.Add("Model", GetType(String))
            tbl.Columns.Add("ProductNumber", GetType(Integer))


            For i As Integer = 0 To RowCount - 1
                Dim rnd As Integer = (New Random(i)).Next(1, 4)
                tbl.Rows.Add(New Object() { i, String.Format("Vendor {0}", rnd), String.Format("Model {0}", i), i * (New Random()).Next(1, 10000) })
            Next i

            Return tbl
        End Function

    End Class
End Namespace
